cmd/kyma/alpha/version/cmd.go

```diff
@@ -0,0 +1,89 @@
+package version
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/metadata"
+	"github.com/kyma-project/cli/cmd/kyma/version"
+	"github.com/kyma-project/cli/internal/cli"
+	"github.com/kyma-project/cli/internal/kube"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type command struct {
+	opts *Options
+	cli.Command
+}
+
+//NewCmd creates a new kyma command
+func NewCmd(o *Options) *cobra.Command {
+
+	cmd := command{
+		Command: cli.Command{Options: o.Options},
+		opts:    o,
+	}
+
+	cobraCmd := &cobra.Command{
+		Use:   "version",
```

Comment:
Please run `make docs` to generate the related documentation files for this command.

Reply:
Ah, cool, didn't know about it

Reply:
Done